src/rcc.rs

```diff
@@ -470,9 +476,9 @@ impl Clocks {
     /// Returns true if the PLL48 clock is within USB
     /// specifications. It is required to use the USB functionality.
     pub fn is_pll48clk_valid(&self) -> bool {
-        // USB specification allow +-0.25%
+        // USB specification allows +-0.25%
         self.pll48clk
-            .map(|freq| (48_000_000 - freq.0 as i32).abs() <= 48_000_000 * 25 / 10000)
+            .map(|freq| (48_000_000 - freq.0 as i32).abs() <= 120_000)
```

Comment:
Why doing this change ? Better to explicit this value by adding to the above comment "i.e. +-120kHz"

Reply:
Because I find it more readable than the weird integer formula.